Platform/Shared/VMWizardState.swift

```diff
@@ -324,7 +330,11 @@ enum VMWizardOS: String, Identifiable {
             }
         }
         if !isSkipDiskCreate {
-            config.drives.append(UTMAppleConfigurationDrive(newSize: storageSizeGib * bytesInGib / bytesInMib))
+            var newDisk = UTMAppleConfigurationDrive(newSize: storageSizeGib * bytesInGib / bytesInMib)
+            if #available(macOS 14, *), useNvmeAsDiskInterface {
+                newDisk.isNvme = true
```

Comment:
Why do we need to remember that this disk is NVMe? Is it a ground work for an ability to add NVMe disks?

Reply:
Nevermind, I see you have already UI for NVMe vs virtio.

Reply:
An option is needed per disk, even without a UI switch. Changing the interface for a disk without considering its current state may break existing VMs and render them unbootable, as the boot drive has changed. Having this ensures that nvme is only used on disks explicitly set to nvme by the user or on newly created VMs.

Reply:
You're right, I've just experienced this myself. I wonder how to raise awareness of what are the benefits and downsides of using NVMe. Maybe there could be more info next to the checkbox?